src/organisms/home/Venue.vue

```diff
@@ -1,7 +1,71 @@
 <template>
-  <div>
+  <div id="venue-background">
     <div class="container">
-      Venue
+      <PageHeading :display="5">Location</PageHeading>
+      <div class="location">
+        <SubPageHeading
+          ><img
+            alt="🇸🇰"
+            src="https://s.w.org/images/core/emoji/11/svg/1f1f8-1f1f0.svg"
+          /><b>Kosice, Slovakia</b></SubPageHeading
+        >
+      </div>
+      <SubPageHeading>
+        <b>Venue:</b> Kunsthalle/Hala umenia Košice
+      </SubPageHeading>
+      <MapViewer
+        :position="{ lat: 48.72483, lon: 21.261012 }"
```

Comment:
Might be better if you move complex objects to computed properties, what do you think?

Reply:
done